// This file is to test the if else structure within a try catch block, it should not produce an error and output nothing

void main () {
    try {
        if (true) {
            int a = stoi("2");
        }
        else{
            int a = stoi("abc");
        }
    }

    catch{
        print("ERROR");
    }
}